### PR TITLE
Pavel admins for approvals

### PR DESCRIPTION
**Pull request by 2HEADSs**
adding functionality for admins waiting for approval
create example function checking admin- only for admins request

### CODE REVIEW

File: src/controllers/userController.js [hunk 1]
```diff
@@ -216,6 +217,18 @@ userController.get('/getMyFavourites', async (req, res) => {
   }
 });
 
+userController.get('/adminsForApproval', async (req, res) => {
+  try {
+    const requesterId = req.requester._id;
+    const result = await getAllAdminsForApprovals(requesterId);
```
Comment (Getsov) [inline src/controllers/userController.js:223]: getAllAdminsForApproval*

File: src/services/userService.js [hunk 2]
```diff
@@ -267,8 +267,32 @@ async function getUserById(userId, requesterId) {
...
+    throw new Error('Your profile is deleted!');
+  }
+  if (!requester.isApproved) {
+    throw new Error('Your profile is not approved!');
+  }
+  if (requester.role !== 'admin') {
+    throw new Error('You do not have access to these records!');
+  }
+  const waitingAdmins = await User.find({
+    isApproved: false,
```
Comment (Getsov) [inline src/services/userService.js:285]: what about deleted admins with isApproved false ?

File: src/utils/adminsCheck.js [hunk 3]
```diff
@@ -0,0 +1,15 @@
+module.exports.checkAdmin = async function (requester) {
```
Comment (Getsov) [inline src/utils/adminsCheck.js:1]: where is this function used?

Comment (2HEADSs) [reply src/utils/adminsCheck.js:1]: it will be used in future development

File: src/services/userService.js (hunk 2 above)
Comment (2HEADSs) [reply src/services/userService.js:285]: fixed

File: src/controllers/userController.js (hunk 1 above)
Comment (2HEADSs) [reply src/controllers/userController.js:223]: fixed + fixed:
organizersForApproval => getAllOrganizersForApproval

**Review by Getsov — APPROVED** [review]

**Pull request merged**